pl/reference/tutorial.rst

```diff
@@ -507,20 +507,19 @@ Receiving data from the form and storing them in the table is the next step.
         }
     }
 
-We then instantiate the Users class, which corresponds to a User record. The class public properties map to the fields
-of the record in the users table. Setting the relevant values in the new record and calling :code:`save()` will store the data in the database for that record. The :code:`save()` method returns a boolean value which indicates whether the storing of the data was successful or not.
+Następnie inicjujemy klasę Users, która odnosi się do rekordu User. Publiczne atrybuty klasy mapują do pól rekordu w tabeli Users. Ustawienie stosownych wartości w nowym rekordzie oraz wywołanie metody :code:`save()` zachowa dane w bazie danych dla tego rekordu. Metoda :code:`save()` zwraca wartość boolean, która wskazuje czy zapis danych był zakończony sukcesem, czy nie.
 
-The ORM automatically escapes the input preventing SQL injections so we only need to pass the request to the :code:`save()` method.
+ORM automatycznie konwertuje specjalne znaki HTML dla pól formularza chroniąc przed wstrzykiwaniem kodu SQL, więc musimy tylko przesłać dane do metody :code:`save()`.
```

Comment:
W oryginale nic nie ma o żadnych znakach html dla pól formularza. Tu chodzi o to że korzysta z bindingu parametrów z automatu z odpowiednim typem danych pobieranym z metadaty. Nie wiem jak do końca przetłumaczyć escapes w tym kontekście, nie możemy po prostu użyć czegoś w stylu `automatycznie oczyszcza dane wejściowe chroniąc przed wstrzykiwaniem` ?